packages/app-bridge/src/global.d.ts

```diff
@@ -2,4 +2,14 @@
 
 type Nullable<T> = T | null;
 
+/**
+Type that behaves just like RequiredExactlyOne from type-fest with only difference that the remaining keys are set to optional.
+
+@category Object
+*/
+type RequireOnlyOne<ObjectType, KeysType extends keyof ObjectType = keyof ObjectType> = {
```

Comment:
Isn't it `RequireAtLeastOne`?

Reply:
Nope, as  RequireOnlyOne mutates other fields to be optional while RequireAtLeastOne leaves other keys as they are.